messages/run.js

```diff
@@ -5,15 +5,10 @@ module.exports = {
 	You can choose the format of output and decide between printing the results directly
 	or as contents of a file that you provide with --outfile flag.`,
 	"flags": {
-		"rulenameDescription": "[description of 'rulename' parameter]",                   // TODO: Change this once the flag is implemented.
```

Comment:
Cleaned up parameters that aren't currently implemented. We can add them back as and when we prioritize their implementation.